ydb_dbapi/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .errors import *  # noqa
```

Comment:
**question (blocking)**: Did you try to configure Ruff to ignore this error in `__init__.py` files?

Reply:
Try this one
```toml
[tool.ruff.lint.per-file-ignores]
"**/test_*.py" = ["S", "SLF", "ANN201", "ARG", "PLR2004"]
"__init__.py" = ["F401"]
```